app/src/main/res/values/colors.xml

```diff
@@ -1,19 +1,40 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <color name="black">#FF000000</color>
-    <color name="gray">#FF555555</color>
-    <color name="light_gray">#FF999999</color>
     <color name="white">#FFFFFFFF</color>
-    <color name="red">#FFFF0000</color>
+    <!--<color name="gray">#FF555555</color>
+    <color name="light_gray">#FF999999</color>
+    <color name="red">#FFFF0000</color>-->
     <color name="purple_200">#FFBB86FC</color>
     <color name="purple_500">#FF6200EE</color>
     <color name="purple_700">#FF3700B3</color>
     <color name="teal_200">#FF03DAC5</color>
     <color name="teal_700">#FF018786</color>
 
-    <color name="amaranth_red">#FFD90429</color>
-    <color name="imperial_red">#FFEF233C</color>
-    <color name="alice_blue">#FFEDF2f4</color>
-    <color name="manatee">#FF8D99AE</color>
-    <color name="space_cadet">#FF2B2D42</color>
+    <!--<color name="amaranth_red">#FFD90429</color>
+     <color name="imperial_red">#FFEF233C</color>
+     <color name="alice_blue">#FFEDF2f4</color>
+     <color name="manatee">#FF8D99AE</color>
+     <color name="space_cadet">#FF2B2D42</color>-->
+
+    <!-- test -->
+    <!--<color name="primary">#FF5722</color>
+    <color name="primary_dark">#E64A19</color>
+    <color name="primary_light">#FFCCBC</color>
+    <color name="accent">#CDDC39</color>
+    <color name="primary_text">#212121</color>
+    <color name="secondary_text">#757575</color>
+    <color name="icons">#FFFFFF</color>
+    <color name="divider">#BDBDBD</color>-->
+
+    <!-- test -->
+    <!--<color name="primary">#5794B7</color>-->
+    <color name="primary">#01b2FA</color>
```

Comment:
Why did you rename the colors this way? Is it by convenience to then change the values of the hexadecimal color values!

Reply:
Yes, it is also more common to name colors in this way because it corresponds more to a color palette. And this allows you to adapt the colors more easily everywhere by simply changing these values.